.github/workflows/release.yml

```
@@ -0,0 +1,222 @@
+              changelog="Release $version - See CHANGELOG.md for full details"
+            fi
+          else
+            changelog="Release $version"
+          fi
+          
+          # Save changelog to file for GitHub release
+          echo "$changelog" > release-notes.md
+          echo "Changelog extracted for $version"
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)